cosmos-tx/tests/integration.rs

```diff
@@ -99,9 +99,26 @@ fn msg_send() {
                 panic!("deliver_tx failed: {:?}", tx_commit_response.deliver_tx);
             }
 
-            // TODO(tarcieri): look up transaction by hash and test transaction parsing
-        });
+            // TODO(tarcieri): avoid race here between when API reports TX available
+            time::sleep(Duration::from_secs(1)).await;
```

Comment:
It seems as though even if `DeliverTx` has succeeded the transaction is sporadically missing from `/tx_search`. Perhaps we need to wait until the next block here?